src/web/src/pages/CoursePage.vue

```diff
@@ -70,10 +62,25 @@ export default {
     this.ready = true;
   },
   computed: {
-    prereqTree() {
-      let treeData = {};
-      treeData.name = this.courseName;
-      return treeData;
+    transformed() {
+      let precoreqtext = this.courseObj.raw_precoreqs;
+      if (precoreqtext === null) {
+        return "This course has no prerequisites";
+      }
+      const regex = /([A-Z]){4}( )([0-9]){4}/g;
+      while (precoreqtext.search(regex) != -1) {
+        let index = precoreqtext.search(regex);
+        let beforetext = precoreqtext.slice(0, index);
+        let dept = precoreqtext.slice(index, index + 4);
+        let cn = precoreqtext
+          .slice(index, index + 9)
+          .split(" ")
+          .join("-");
+        let link = '<a href="/explore/'.concat(dept, "/", cn, '">', cn, "</a>");
```

Comment:
Prefer if we can use `router-link`

Reply:
I was trying to use router link but it didn't seem to like it with the vthml in the template. I think this is because the vbind:to directive doesn't work inside it. After I tried using dynamic components but that was a whole other mess. There might be a way to get it to work but I couldn't figure it out and the a tags worked?

Reply:
Yeah looks like vhtml doesnt work well with components. The only issue with `a` links is they will reload the page so if we start storing state in vuex this is gonna reset it. I guess there's no good solution here so we'll just keep it as it is